odk1-src/collect-offline-maps.rst

```diff
@@ -1,44 +1,68 @@
+#. Select your offline layer in the :ref:`reference layer settings <reference-layer-settings>`.
+#. Open a :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, or :ref:`geoshape <geoshape-widget>` questions.
+
+Your offline layer should be displayed. If it has transparency (PNG or PBF tiles only), the selected basemap will show through. If it does not have transparency or you are offline, only your offline layer will be displayed.
 
 .. _getting-map-tiles:
 
-Getting map tiles
+Getting map tilesets
 -------------------------
-`Open Map Tiles <https://openmaptiles.org/>`_ hosts many free map tile files that can be used in ODK Collect.
+`OpenMapTiles <https://openmaptiles.org/>`_ hosts many free map tile files that can be used in Collect.
 
-To create MBTiles files, use one of the `compatible applications <https://github.com/mapbox/mbtiles-spec/wiki/Implementations#applications>`_ that can write tiles. Commonly used free software packages are `TileMill <https://tilemill-project.github.io/tilemill/>`_ and `QGIS <https://qgis.org/en/site/>`_ with the `QTiles plugin <https://github.com/nextgis/QTiles#qtiles>`_.
+To create MBTiles files, use one of the `compatible applications <https://github.com/mapbox/mbtiles-spec/wiki/Implementations#applications>`_ . Commonly used free software packages are `TileMill <https://tilemill-project.github.io/tilemill/>`_ and `QGIS <https://qgis.org/en/site/>`_ with the `QTiles plugin <https://github.com/nextgis/QTiles#qtiles>`_.
 
+If you have existing geospatial data that is not in an MBTiles file, you may be able to convert it for use in Collect. For example, `Tippecanoe <https://github.com/mapbox/tippecanoe>`_ is a tool to build vector MBTiles files from GeoJSON features (see warning above: vector MBTiles files are only supported with Mapbox basemaps and are displayed without styling).
+
+The name of the tileset provided in the MBTiles file will be displayed in the Collect layer selection menu. The filename will also be displayed.
 
 .. _transferring-offline-tiles:
 
-Transferring offline tiles to devices
--------------------------------------
-To make MBTiles files available for use in ODK Collect they must be manually transferred to Android devices. The tile files need to be inside a folder in the :file:`/sdcard/odk/layers` folder. The folder name will be used to identify your offline map in the user interface so choose a user-friendly name for it. For example, if your MBTiles file is `my_tiles.mbtiles` and it includes information about settlements in the mapped area, you may want to call your folder :file:`/sdcard/odk/layers/All settlements/`.
+Transferring offline tilesets to devices
+-----------------------------------------
+MBTiles files must be manually transferred to Android devices to be available to Collect. Place the MBTiles files in the :file:`/sdcard/odk/layers` folder.
```

Comment:
Maybe redundant, but can't hurt?

    ...folder, and ensure their filenames end in `.mbtiles`.